bootstrap_test.py

```diff
@@ -280,6 +280,33 @@ def test_consistent_range_movement_true_with_rf1_should_fail(self):
     def test_consistent_range_movement_false_with_rf1_should_succeed(self):
         self._bootstrap_test_with_replica_down(False, rf=1)
 
+    def test_rf_gt_nodes_multidc_should_succeed(self):
```

Comment:
We could add some brief docstring mentioning the purpose of the test and mentioning the related ticket(s) with `@jira_ticket`.